solidity/test/token_stake/TestTokenStake.js

```diff
@@ -557,12 +592,13 @@ contract('TokenStaking', function(accounts) {
     })
   
     it("should report no active stake before initialization period is over", async () => {
-      await delegate(operatorOne, stakingAmount)
+      let tx = await delegate(operatorOne, stakingAmount)
+      let createdAt = (await web3.eth.getBlock(tx.receipt.blockNumber)).timestamp
       await stakingContract.authorizeOperatorContract(
         operatorOne, operatorContract, {from: authorizer}
       )
   
-      await mineBlocks(initializationPeriod - 1)
+      await increaseTimeTo(createdAt + initializationPeriod - 1)
```

Comment:
[This](https://github.com/keep-network/keep-core/pull/1512/files/d339004f82283c04495c8638676b7e87b08fe6c2#r397994846) comment still applies. I think we should have `await increaseTimeTo(createdAt + initializationPeriod)` here. This is just before the initialization period is over.